index.js

```diff
@@ -113,4 +117,76 @@ app.get('/squadStats', async (req, res) => {
 
     res.send(retVal);
 
-});
\ No newline at end of file
+});
+
+//stats squad individual
+app.get('/squadStatsIndiv', async (req, res) => {
+
+    let gt = req.query.gt;
+
+    let result = await lib.halo.mcc['@0.0.11'].squad.activity({
+        gamertags: [
+            'sagwasswagga',
+            'Cragle 3',
+            'A Pregnant Nunn',
+            'babaganoosh069',
+            'Girthy Jello',
+            'SirPuffingtin',
+            'ya boi nar',
+            'l swordzzz'
+        ],
+        minSquadSize: '3',
+        game: 'Halo 3'
+    });
+
+
+    let games = result.games;
+
+    let aggregSqdStats = [];
+    
+    for(let match of games) {
+        let win = match.squadWon;
+        let avgHeadshotRate = 0.0;
+        let avgKD = 0.0;
+        let avgScore = 0.0;
+        let avgKills = 0.0;
+        let avgDeaths = 0.0;
+        let avgAssists = 0.0;
+        let playedAt = match.playedAt;
+
+        for(let stat of match.stats) {
+            if(stat.gamertag.toUpperCase() == gt.toUpperCase()) {
+                console.log(stat)
+                //aggregate them all here
+                avgHeadshotRate+=(stat.headshotRate);
+                avgKD+=(stat.killDeathRatio);
+                avgScore+=(stat.score);
+                avgKills+=(stat.kills);
+                avgDeaths+=(stat.deaths);
+                avgAssists+=(stat.assists);
+            }
+        }
+
+        let aggStat = {
+            win: win,
+            avgHeadshotRate: avgHeadshotRate,
+            avgKD: avgKD,
+            avgScore: avgScore,
+            avgKills: avgKills,
+            avgDeaths: avgDeaths,
+            avgAssists: avgAssists,
+            playedAt: playedAt
+        };
+        
+        aggregSqdStats.push(aggStat);
+        
+    }
+
+
+    retVal = {
+        data: aggregSqdStats
+    };
+
+    res.send(retVal);
```

Comment:
where does this get sent to

Reply:
this gets sent back to the application that issued the request, in our case it will be our discord bot/website/frontend that we decide on